_includes/v2.0/prod-deployment/insecure-start-nodes.md

```diff
@@ -130,6 +130,7 @@ For each initial node of your cluster, complete the following steps:
      Flag | Description
      -----|------------
      `--join` | Identifies the address and port of 3-5 of the initial nodes of the cluster.
+     `--advertise-host` | The hostname or IP address to advertise to other CockroachDB nodes. If it is a hostname, it must be resolvable from all nodes; if it is an IP address, it must be routable from all nodes.
```

Comment:
I'm in the process of updating this and other text around --advertise-host in a separate PR. I can just resolve the conflict on my end, or you can revert this change. Your call.